app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,23 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    # image.openscap_result.try(:destroy)
```

Comment:
I see that you want to un-comment this code once 
> a new image_inspector_client is available. See also this comment

which is fine.
But how about moving this bit to a subsequent PR? Maybe the gem never gets ready or the api changes?

At least I would expect some explanation inside the code why this is commented out.